src/test/scala/com/cognite/sdk/scala/v1/TransformationsTest.scala

```diff
@@ -261,4 +261,39 @@ class TransformationsTest extends CommonDataModelTestHelper with RetryWhile {
                                      |}""".stripMargin
   }
 
+  it should "run transformation and get job by id" in {
+    val uniquePrefix = shortRandomUUID()
+
+    val newTransformation = TransformationCreate(
+      s"$uniquePrefix-transformation-sdk-test",
+      Some(s"select 1"),
+      Some(GeneralDataSource("events")),
```

Comment:
maybe call this `GenericDataSource` or just `DataSource`?

Reply:
I rename it to GenericDataSource, we have DataSource in jetfire-backend as well so less confusion ;)